app/views/restaurants/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Restaurants</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Genre</th>
+      <th>Rating</th>
+      <th>Tenbis</th>
+      <th>Address</th>
+      <th>Delivery time</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @restaurants.each do |restaurant| %>
+      <tr>
+        <td><%= restaurant.name %></td>
```

Comment:
The restaurant part should probably be moved to a partial. Think about template structure when doing the next part of the project, to make files easier to read.